app/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$DATABASE" = "postgres" ]
+  then
+    echo "Waiting for psql"
+
+    while ! nc -z $SQL_HOST $SQL_PORT; do
+      sleep 0.1
+    done
+
+    echo "Psql started"
+fi
+
+python app/manage.py makemigrations
```

Comment:
this is not correct. you must create migrations only manually

Reply:
ok, done